servatrice/src/servatrice.cpp

```diff
@@ -543,27 +549,51 @@ void Servatrice::statusUpdate()
     servatriceDatabaseInterface->execSqlQuery(query);
 
     // send activation emails
```

Comment:
Comment is outdated now I think. But this kind of comment hints that the code following may want to be extracted to a method with a similar name as the comment text